docs/changelog.rst

```diff
@@ -229,6 +229,7 @@ Bug fixes:
   already. A new option ``--noinherit/-I`` to :ref:`modify <modify-cmd>`
   allows changing this behaviour.
   :bug:`4822`
+* Fix bug where an interrupted import process poisons the database.
```

Comment:
LOL, nicely worded! I like! You could add that the fix is about the path field in particular.